### PR TITLE
add generic cloud formation

### PR DESCRIPTION
**Pull request by iakov-aws**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: bump-release.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+bump='patch'
+if len(sys.argv)>1 :
+	bump = sys.argv[1]
+
+maj, minor, patch = map(int, old_ver.split('.'))
+
+if bump=='patch':
+	new_ver = '.'.join(map(str,[maj,minor,patch + 1]))
+elif bump=='minor':
+	new_ver = '.'.join(map(str,[maj,minor + 1,patch]))
```
Comment (darken99) [inline bump-release.py:15]: If we bump minor or major version number we have to set lower numbers to 0

Comment (iakov-aws) [reply bump-release.py]: yes good catch

File: cfn-templates/cid-cfn.yml [hunk 2]
```diff
@@ -0,0 +1,1116 @@
...
+    Condition: NeedRefreshDatasets
+    Properties:
+      FunctionName:  !Sub 'CidSpiceRefreshLambda${Suffix}'
+      Role: !GetAtt SpiceRefreshExecutionRole.Arn
+      Description: 'Function for CID datasets refresh'
+      Runtime: python3.9
+      Architectures: [ arm64 ]
+      MemorySize: 128
+      Timeout: 60
+      Handler: 'index.lambda_handler'
```
Comment (tbuatois) [inline cfn-templates/cid-cfn.yml:190]: Duplicate handler at line 195

File: cfn-templates/cid-cfn.tests.bats [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+      DeployKPIDashboard="yes"\
+      DeployTAODashboard="yes"\
+      AthenaWorkgroup=""\
+      AthenaQueryResultsBucket=""\
+      CURDatabaseName=""\
+      CURPath=""\
+      CURTableName=""\
+      CidVersion="$cid_version"\
+      QuickSightDataSetRefreshSchedule="cron(0 4 * * ? *)"\
+      QuicksightIdentityRegion="us-east-1"\
```
Comment (tbuatois) [inline cfn-templates/cid-cfn.tests.bats:79]: add region from credentials (aka. where quicksight is deployed)
export region=$(python3 -c 'import boto3; print(boto3.Session().region_name)')
QuicksightIdentityRegion="$region"\

**Review by darken99 — APPROVED** [review]

File: cfn-templates/cid-cfn.tests.bats (hunk 3 above)
Comment (iakov-aws) [reply cfn-templates/cid-cfn.tests.bats:79]: will be in the next release

**Pull request merged**